src/utils/types.ts

```diff
@@ -18,13 +18,24 @@ export type OptimisedSVG = {
     height: string;
   };
 };
-export type Icons = { [name: string]: OptimisedSVG };
+export type Icons = { [name: string]: { $value: OptimisedSVG } };
```

Comment:
Are these the same as a `Dictionary` that we have also used in a few places? Might be good to make consistent if so

Reply:
oo yeah nice catch, hmm what would be a good name for it? `nestedValue`?